File: samples/src/main/java/jvmMain/java/safe/Withdrawal.java
package jvmMain.java.safe;

import jvmMain.java.Common;
import jvmMain.kotlin.Config;
import one.mixin.bot.safe.SafeException;
import one.mixin.bot.safe.TransactionKt;
import one.mixin.bot.vo.safe.TransactionResponse;

import java.io.IOException;
import java.util.List;
import java.util.UUID;

public class Withdrawal {
    public static void main(String[] args) {
        String traceId = UUID.randomUUID().toString();
        try {
            List<TransactionResponse> transactions = TransactionKt.withdrawalToAddress(
                    Common.botClient,
                    Common.Token.TRON_USDT.getAssetId(),
                    "TQ5NMqJjhpQGK7YJbESKfjXZoQXf2hZL7r",
                    null,
                    "0.0001",
                    null,
                    traceId
            );
            for (TransactionResponse transaction : transactions) {
                System.out.println("view in view transaction in: https://viewblock.io/mixin/tx/" + transaction.getTransactionHash());
            }
        } catch (SafeException | IOException e) {
            throw new RuntimeException(e);
        }
    }

}

File: samples/src/main/java/jvmMain/kotlin/safe/Withdrawal.kt
package jvmMain.kotlin.safe

import jvmMain.kotlin.Token
import jvmMain.kotlin.botClient
import one.mixin.bot.safe.withdrawalToAddressBlocking
import java.util.UUID

fun main() {
    val traceId = UUID.randomUUID().toString()
    val transactions = withdrawalToAddressBlocking(
        botClient = botClient,
        assetId = Token.TRON_USDT.assetId,
        amount = "0.003",
        memo = "test-memo-from-kotlin",
        destination = "TAXE98CMomoA28MtNpfxUutCBq2i4bDbRv",
        tag = null,
        traceId = traceId,
    )
    for (tx in transactions) {
        println("view transaction in: https://viewblock.io/mixin/tx/${tx.transactionHash}")
    }
}
